### DISCUSSION TITLE
TuringCourses/math_phys/probability/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#52] [General]
# TuringCourses/math_phys/probability/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/probability/

**Comment by ZhouTimeMachine** [comment]
推荐书单是 lhjgg 写的，我一本都没看过（瘫），如果只想混过这门课不必看

**Comment by Fairicle** [comment]
如果仔细看了历年卷以及数院概率论的历年卷（在 98 上面能够找到），可以发现试卷大致有两种风格：一种是不怎么绕弯子，通向答案的路径很直接，比较强调计算，风格和作业题比较相似，如 21 级历年卷；第二种是大部分题目都需要拐个弯，你可能需要一点技巧（如果数分学的好的话是有帮助的），否则有可能第一问就被卡住，如 22 级历年卷（当然，也可能单纯是 21 级的卷子因为疫情出的简单）。

说这些是因为不少 22 级的同学因为这门课平时压力不大、专业课缠身等等而忽视了概率论的学习，远没有学数分线代的时候认真，草草做了几份历年卷便上考场而被创死了（悲）。这门课虽然没有数分线代困难，但是也不是考前突击便能拿到好看的分数的。请大家不要完全放弃平时对概率论的学习，至少每周的作业要认真做，以及原理性的东西一定要掌握，这样每道题不管算不算得出来，至少知道算的是什么、大致要怎么算，把过程写上就能拿到不少的分数了。

**Comment by VoteVeto1** [comment]
我个人加几本推荐书，推荐书单第二本开始就是measure based的施利亚耶夫强度有点太高了

\bm{Introduction to Probability Theory} by  Bertsekas and Tsitsiklis： MIT 工科概率论教材，打基础的古典概率部分特别好，有一些markov and poisson process引入，在你正式接触随机过程（又名概率论II）之前一个很好的intro。

\bm{概率论}, 应坚刚和何萍：旦子的理科概率论教材，写的最好的内容是\bm{随机向量}这一章。习题质量极高，让你深刻体会到什么叫做抛离低级趣味的概率论习题，且很多习题都在为你后续《数理统计》习题做铺垫。可能是全世界最喜欢讲Cauchy分布的书：这个概率世界可能唯一重要的反例。

\bm{概率论}，何书元：北大黄皮，可以直接去看条件期望/母函数和随机变量序列收敛这三部分。讲的最好的是最后的进阶内容。

\bm{Probability: Theory and Examples (PTE)} by Rick Durrett: 北美概率宗师。这本书是为有志于进一步在研究生/博士生阶段的未来科研人才准备的。
